.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn run lint-staged
```

Comment:
This file needs to be marked as executable (`chmod +x .husky/pre-commit`).  I will do this.